website_sale_product_pack/__manifest__.py

```diff
@@ -4,10 +4,15 @@
     "name": "Website Sale Product Pack",
     "category": "E-Commerce",
     "summary": "Compatibility module of product pack with e-commerce",
-    "version": "13.0.1.0.2",
+    "version": "16.0.1.0.3",
```

Comment:
```suggestion
    "version": "16.0.1.0.0",
```

Reply:
ping @lef-adhoc

Reply:
@ioans73 I have already made the change, is it ok?

Reply:
@pedrobaeza If you could review again...